src/plugins/kibana_utils/public/history/get_query_params.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { parse, ParsedQuery } from 'query-string';
+import { Location } from 'history';
+
+export function getQueryParams(location: Location): ParsedQuery {
```

Comment:
Looks very similar to the utility in src/plugins/es_ui_shared/public/url/extract_query_params.ts. Not sure the best approach here but it would be nice if we could combine these.

Reply:
Thanks for point me to it! 
I extracted my function from existing `history` helper (in kibana_utils) so technically I didn't introduce new code.
I decided to leave as is keeping dashboard -> kibana_utils dep and not introducing dashboard -> es_ui dep. 

Probably ideally `es_ui_shared/public/url/extract_query_params.ts` should be removed and utility from `kibana_utils` should be used instead, but `es_ui_shared` is not dependent on `kibana_utils` today, so doesn't worth introducing dependency to maintain for these couple lines.